lib/webrick-patch.rb

```diff
@@ -0,0 +1,36 @@
+require 'webrick/https'
+
+CIPHERS = 'AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256'
+SSL_VERSION = 'TLSv1_2_server'
```

Comment:
> Could you post a traceback?

<pre>
D, [2015-12-01T15:21:21.188543 #23426] DEBUG -- : unknown SSL method `TLSv1_2_server'.:/usr/share/foreman-proxy/lib/webrick-patch.rb:18:in `ssl_version='
/usr/share/foreman-proxy/lib/launcher.rb:125:in `join'
/usr/share/foreman-proxy/lib/launcher.rb:125:in `launch'
/usr/share/foreman-proxy/bin/smart-proxy:6
</pre>